canonical-websites/vanillaframework.io
# Update to use latest vanilla-brochure-theme

## Done
Update the gulp files to the latest use. Changed the vanilla dependency to vanilla-brochure-theme.

## QA
- Run through the updated Readme to set up and check it all works correctly
- Check the site loads as before

## Details
Fixes https://github.com/canonical-websites/vanillaframework.io/issues/7
